ipaclient/install/client.py

```diff
@@ -2166,7 +2169,7 @@ def install_check(options):
             "You must be root to run ipa-client-install.",
             rval=CLIENT_INSTALL_ERROR)
 
-    tasks.check_selinux_status()
+    selinux_works = tasks.check_selinux_status()
```

Comment:
I guess check_selinux_status() will be called from RedHatTaskNamespace, right?  otherwise: NotImplementedError() will be raised.

Reply:
thanks abbra, that clarifies.